app-starters-common/pom.xml

```diff
@@ -24,4 +24,12 @@
 		<module>app-starters-twitter-common</module>
 	</modules>
 
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-java-dsl</artifactId>
+			<version>${spring-integration-java-dsl.version}</version>
+		</dependency>
```

Comment:
If we will based on the `spring-boot-starter-integration`, we should consider to remove this dep from here. 
Since Boot-1.4.0 SI-Java-DSL is always included.